src/app/ipv/middleware.js

```diff
@@ -480,6 +480,31 @@ module.exports = {
       next(error);
     }
   },
+  formHandleCoiDetailsCorrect: async (req, res, next) => {
+    try {
+      const { context, currentPage } = req?.session || "";
```

Comment:
Ah one thought here is I believe we can't destructure from the string, so maybe the default should be `{}`. I know it's used elsewhere in core-front but there's a ticket to clear it up, so you could let it be cleared up by [PYIC-5606](https://govukverify.atlassian.net/browse/PYIC-5606)